src/routes/subscribe/addSubscriber.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from 'express';
+import Subscriber from '../../models/Subscriber';
+import { validateEmail } from '../../utils/validateEmail';
+
+const addSubscriber = async (req: Request, res: Response) => {
+    try {
+        const { email } = req.body;
```

Comment:
lowercase the email here (before checking it with existing emails in db since they all are lowercased)